fxa-auth-db-server/index.js

```diff
@@ -117,6 +121,18 @@ function createServer(db) {
 
   api.get('/__heartbeat__', reply(db.ping))
 
+  api.del(
+    '/account/:uid/device/:deviceId',
+    function (req, res, next) {
+      db.deleteDevice(req.params.uid, parseInt(req.params.deviceId, 10) || 0)
+        .then(
+          handleSuccess.bind(null, req, res),
+          handleError.bind(null, req, res)
+        )
+        .done(next, next)
+    }
+  )
```

Comment:
So having read further down, I see why you want to generate the ids internally here.  It still feels a bit out-of-place given the rest of the API, but perhaps it's the only reasonable way to get the combination of short-enough-to-not-be-identifiable and guaranteed-not-to-collide-per-user.

We should carefully consider what we're guarding against by making the device-ids short though.  If they only ever appear on the wire alongside a long-and-uniquely-identifiable sessionToken id, it may not provide much value.